### ISSUE TITLE
Cleaning out the Roof Gutters

### ISSUE THREAD

**Issue by iteles** [dwyl/home#299]
As we know, rainy season is coming and our gutters are blocked with Autumn leaves both from the trees on the sidewalk infront of the house and also our own https://github.com/dwyl/home/issues/298

This is a HIGH priority as it will cause infiltrações in our house if they're not cleared (and Carlos confirmed they're full to the brim).

I have called 2 companies for quotes and both have a €20+IVA deslocação charge to give a quote:

## 1. [Limpa-chaminé](https://www.limpa-chamine.pt/desentupimentos-de-caleiras-braga-sao-vicente-braga/)
963767155
These guys have told us it's a minimum of €180+IVA MINIMUM but if they can do the work and we agree to the quote they can do it on the spot.
***I have booked for them to come and give a quote on Tuesday at 09h30*** but I can change this if not a good time.

## 2. [Telesgotos](https://www.telesgotos.pt/servicos-telesgotos-desentupimentos/)
These were the guys who unblocked the drain infront of our garage. They weren't cheap but they were ok and got the job done! 
They weren't able to give me a minimum quote but are available from Tuesday on.

Should I just cancel the first guys and ask for a quote from the ones we already know?
And yes, I know the key now is to unblock the entrance to the roof.

**Comment by nelsonic** [comment]
@iteles on a _dry_ day I don't mind going up there with my safety gear (fall arrest, grippy shoes & helmet). 👷‍♂️ 💭 
But at the very least we need to fly the drone over the roof to assess the level of blockage. 🛸

**Comment by iteles** [comment]
Does this mean I should cancel the Tuesday quote visit?

**Comment by nelsonic** [comment]
No. Don't cancel. But if it's raining between now and then, will they even show up to quote? 🤷‍♂️

**Comment by nelsonic** [comment]
@iteles it's sunny/mild today. ☀️ 
Want to fly the drone over the gutters _before_ the cleaning person arrives for quote? 💭
